Loop/Settings/MoreSettingsView.swift

```diff
@@ -59,13 +59,13 @@ struct MoreSettingsView: View {
             })
 
             Section("Stage Manager") {
-                Toggle("Respect Stage Manager", isOn: $respectStageManager)
+                Toggle("Respect stage manager", isOn: $respectStageManager)
 
                 CrispValueAdjuster(
-                    "Stage Strip Size",
+                    .init(localized: .init("Crisp Value Adjuster: State Strip Size", defaultValue: "State strip size")),
```

Comment:
Small typo: "State" should be replaced with "Stage"

Reply:
Updated.